File: app/src/main/java/org/wheatgenetics/androidlibrarybuilder/MainActivity.java
package org.wheatgenetics.androidlibrarybuilder;

/**
 * Uses:
 * android.os.Bundle
 * android.support.v7.app.AppCompatActivity
 *
 * org.wheatgenetics.androidlibrarybuilder.R
 */
public class MainActivity extends android.support.v7.app.AppCompatActivity
{
    @java.lang.Override protected void onCreate(final android.os.Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        this.setContentView(org.wheatgenetics.androidlibrarybuilder.R.layout.activity_main);
    }
}

File: app/src/main/java/org/phenoapps/activities/MainActivity.kt
package org.phenoapps.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import org.phenoapps.androidlibrarybuilder.R

class MainActivity: AppCompatActivity() {

    companion object {
        private var TAG = this::class.simpleName
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)
    }

}
